gym/envs/nysaq/kblocker.py

```diff
@@ -105,15 +107,19 @@ def _reset(self):
         # Initialize the agents randomly
         self.__coords = [(r, 0) for r in random.sample(range(self.__grid_x), self.__num_bs + 1)]
 
-        return np.array(list(map(self.__encode__, self.__coords)))
+        return self._get_encoded_state()
+
+
+    def _get_encoded_state(self):
+        return sum(map(list, self.__coords), [])
```

Comment:
Could you please add a comment here for readability? took me a few minutes to figure out what's happening here.

Reply:
Done.  And refactored the code so it is much simpler!